Doc/library/decimal.rst

```diff
@@ -790,7 +790,7 @@ Decimal objects
       the current thread's context is used.
 
       An error is returned whenever the resulting exponent is greater than
-      :attr:`Emax` or less than :attr:`Etiny`.
+      :attr:`!Emax` or less than :attr:`!Etiny`.
```

Comment:
```suggestion
      :attr:`Emax` or less than :meth:`Etiny`.
```

Fix this better—`Etiny` is a method defined and documented on a `Context`, while `Emax` is a (undocumented) attribute of a `Context`. The latter now points to the correct method, while the former remains a valid warning as a reminder to document it.